### PR TITLE
kubeone: update docs to v1.0.0-beta.2

### PR DESCRIPTION
**Pull request by xmudrii**
/assign @kron4eg

### CODE REVIEW

**Comment by xmudrii** [comment]
/hold
to update the readme

File: content/kubeone/master/_index.en.md [hunk 1]
```diff
@@ -178,9 +178,10 @@ a new feature or improvement, please check out our [contributing guide][2].
 
 If you want to get in touch with us and discuss about improvements and new
 features, please create a new issue on GitHub or connect with us over the
-mailing list or Slack:
+mailing list, forums, or Slack:
 
 * [`#kubeone` channel][4] on [Kubernetes Slack][10]
+* [Kubermatic forums][15]
 * [loodse-dev mailing list][9]
```
Comment (xmudrii) [inline content/kubeone/master/_index.en.md:185]: @kron4eg Should we leave or drop this mailing list?

Comment (kron4eg) [reply content/kubeone/master/_index.en.md:185]: >or drop this mailing list

👍

Comment (xmudrii) [reply content/kubeone/master/_index.en.md]: @kron4eg Done, I'll create a PR to do the same in the KubeOne repo.

File: content/kubeone/master/_index.en.md [hunk 2]
```diff
@@ -178,10 +178,10 @@ a new feature or improvement, please check out our [contributing guide][2].
 
 If you want to get in touch with us and discuss about improvements and new
 features, please create a new issue on GitHub or connect with us over the
-mailing list or Slack:
+mailing list, forums, or Slack:
```
Comment (kron4eg) [inline content/kubeone/master/_index.en.md:181]:
```suggestion
forums or Slack:
```

File: content/kubeone/master/_index.en.md [hunk 3]
```diff
@@ -178,10 +178,10 @@ a new feature or improvement, please check out our [contributing guide][2].
...
 
 * [`#kubeone` channel][4] on [Kubernetes Slack][10]
-* [loodse-dev mailing list][9]
```
Comment (kron4eg) [inline content/kubeone/master/_index.en.md:184]: this [9] can be removed I suppose

```
[9]: https://groups.google.com/forum/#!forum/loodse-dev
```

**Comment by xmudrii** [comment]
@kron4eg Fixed comments, PTAL.

**Pull request merged**